config/script.go

```diff
@@ -370,6 +370,8 @@ func unmarshalScriptCommandOptions(obj cty.Value, expr hhcl.Expression) (*Script
 			}
 			r.CloudSyncPreview = v.True()
 
+		case "layer":
+			fallthrough
 		case "sync_layer":
 			fallthrough
```

Comment:
we just broke the world record for highest number of fallthrough in the same Go file 😂